composer.json

```diff
@@ -87,5 +86,8 @@
                 "Make menu items export properly": "web/patches/menu_link_uuid_nid-2670954-reroll.patch"
             }
         }
+    },
```

Comment:
@LindsayYoung Would you be willing to move this piece to a separate PR? In the interest of separating concerns more clearly.